styles/main.scss

```diff
@@ -0,0 +1,25 @@
+@import './utils/variables';
+
+@import './utils/reset';
+@import './blocks/header';
+@import './blocks/title';
+@import './blocks/logo';
+@import './blocks/navigation';
+@import './blocks/burger';
+@import './blocks/phone';
+@import './blocks/menu';
+@import './blocks/links';
+@import './blocks/main';
+@import './blocks/product';
+@import './blocks/section';
+@import './blocks/category';
+@import './blocks/pictures';
+@import './blocks/recommended';
+@import './blocks/button';
+@import './blocks/about';
+@import './blocks/benefits';
+@import './blocks/contact';
+@import './blocks/form';
+@import './blocks/address';
+@import './blocks/footer';
+@import './blocks/icons';
\ No newline at end of file
```

Comment:
It's a good practice to end your files with a newline. This is a standard convention that's followed by a lot of UNIX/Linux tools. It can help with version control systems and also when concatenating files.